backend/typescript/rest/camperRoutes.ts

```diff
@@ -35,4 +37,43 @@ camperRouter.post("/register", createCamperDtoValidator, async (req, res) => {
   }
 });
 
+/* Get all campers, optionally filter by camp ID */
+camperRouter.get("/campers/", async (req, res) => {
```

Comment:
camper/campers looks a bit funny haha, we should change this to just /campers 
```suggestion
camperRouter.get("/", async (req, res) => {
```